package/cfg/k3s-cis-1.7-hardened/master.yaml

```diff
@@ -285,7 +285,7 @@ groups:
 
       - id: 1.1.19
         text: "Ensure that the Kubernetes PKI directory and file ownership is set to root:root (Automated)"
-        audit: "find /var/lib/rancher/k3s/server/tls | xargs stat -c %U:%G"
+        audit: "stat -c %U:%G /var/lib/rancher/k3s/server/tls"
```

Comment:
This changes the desired check only to check the directory and not all recursive files. The original command is correct. We just need to update the test_items configuration to compare the output and validate that the only strings returned are `root:root` 
So this should pass
```
root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:roo
```

This should not pass.
```
root:root root:root NOT_THE_ROOT_USER:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:root root:roo
```
This can be done with regex

Reply:
After taking another look at this you should be able to leverage this script to accomplish our goal using [check_files_owner_dir.sh](https://github.com/rancher/security-scan/blob/master/package/helper_scripts/check_files_owner_in_dir.sh) similar to what is done in this [test](https://github.com/rancher/security-scan/blob/bb9f870e95d8d709ad090bc2832ba3bc6ee73c8a/package/cfg/rke-cis-1.7-permissive/etcd.yaml#L45-L59)